#151994140 Email and Sms Notification

enodi opened the pull request:
**What does this PR do?**
Send Email and Sms notification to users

**Description of Task to be completed?**
Email and Sms notification is sent when a user makes an order and when shipment status of the order changes (packed, delivered)

**How should this be manually tested?**

-  User makes an order

- Shipping status of order changes

**What are the relevant pivotal tracker stories?**
#151994140

**Screenshots (if appropriate)**
![image](https://user-images.githubusercontent.com/29783644/32246136-4bcaeca4-be7e-11e7-9ac5-18a5b09e1b9f.png)
![image](https://user-images.githubusercontent.com/29783644/32246156-5f1d7a6a-be7e-11e7-8263-b5bf75a48896.png)

enodi committed:
feat(order): implement email notification

- ensure users receive email notifications when an order is placed

[Delivers #151994140]

temilaj commented on server/methods/core/orders.js:
```diff
@@ -427,18 +437,56 @@ Meteor.methods({
+        to: order.email,
+        from: "RAINIER-RC",
+        subject: "Your order is confirmed",
+        // subject: `Order update from ${shop.name}`,
```
why do we have commented out code in source control?

temilaj reviewed (changes requested):
Made a comment here. Please review

temilaj commented:
please squash your new change in to the prevoius commit

enodi committed:
feat(order): implement sms notification

- ensure users receive sms notifications when an order is placed

[Delivers #151994140]
